docker/Dockerfile

```diff
@@ -18,6 +18,7 @@ WORKDIR /app
 # Install Poetry
 RUN pip install poetry==2.1.3
 RUN poetry config virtualenvs.create false
+RUN pip install pytest-asyncio
```

Comment:
@benjigifford I think we should not add that to the base image. This is only used for the tests, right?